apps/site/tests/integration/schemapage.test.ts

```diff
@@ -38,20 +36,20 @@ test("schema page should contain key elements", async ({ page }) => {
 
   await expect(
     page.locator(
-      "text=You can use these entity types as the expected value for a property in another schema.",
+      "text=You can use these entity types as the expected value for a property in another ",
```

Comment:
I appreciate you're just fixing the test but where does this occur. How can an entity type be an expected value for a property?

Reply:
Oh, I think this was a mass replace in the file that went awry. It was probably something to do with links

Reply:
It's looking for copy that doesn't exist and we can clean up when fixing the tests (this PR is just making TS compile)